src/Symfony/Bundle/FrameworkBundle/Tests/Functional/Bundle/TestBundle/Resources/config/routing.yml

```diff
@@ -22,9 +22,9 @@ session_showflash:
     path:     /session_showflash
     defaults: { _controller: Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\Controller\SessionController::showFlashAction }
 
-profiler:
-    path:     /profiler
-    defaults: { _controller: Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\Controller\ProfilerController::indexAction }
+inspector:
+    path:     /inspector
+    defaults: { _controller: Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\Controller\InspectorController::indexAction }
```

Comment:
Shouldn’t we add a redirection from `/profiler` to `/inspector` for 4.4?
I know this is only for dev mode, but some people might consider this as a BC break.